_docs/configuration_files/foremast_config.rst

```diff
@@ -88,6 +88,8 @@ FQDN Of your spinnaker Gate instance. This is where all API calls to Spinnaker w
 
     | *Required*: Yes
 
+.. _templates_path:
```

Comment:
To link to the template_path if they choose to override our defaults we ship with.